manim/_config/main_utils.py

```diff
@@ -469,4 +469,8 @@ def _parse_args_no_subcmd(args: list) -> argparse.Namespace:
         metavar="True/False",
     )
 
+    parser.add_argument(
+        "--jupyter_width", type=str, help="Width of the displayed media on Jupyter"
+    )
+
```

Comment:
I believe this file is responsible for adding the option to the CLI which we don't want since jupyter won't be called from the CLI (I could be wrong).
```suggestion
```